src/components/ReanimatedSwipeable.tsx

```diff
@@ -584,4 +633,10 @@ const styles = StyleSheet.create({
     ...StyleSheet.absoluteFillObject,
     flexDirection: I18nManager.isRTL ? 'row' : 'row-reverse',
   },
+  childrenContainer: {
+    flex: 1,
+    alignItems: 'center',
+    width: '100%',
+    height: '100%',
+  },
```

Comment:
I don't think this is needed here (and it was causing the issue from the screen above)

Reply:
It ensures the element is clickable and draggable across it's entire area, 
but I think you're right suggest to removing it, as the wasn't clickable across the entire area.